doc/reference/configuration/configuration_reference.rst

```diff
@@ -3133,6 +3133,43 @@ log.syslog.*
     | Default: box.NULL
     | Environment variable: TT_LOG_SYSLOG_SERVER
 
+.. _configuration_reference_lua:
+
+lua
+---
+
+The ``lua`` section outlines the configuration parameters related to the Lua environment within Tarantool.
+
+.. NOTE::
+
+    ``lua`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+..  NOTE::
+
+- :ref:`lua.memory <configuration_reference_lua>`
+
+
+.. _configuration_reference_lua_memory:
+
+.. confval:: lua.memory
+
+Specifies the maximum memory amount available to Lua scripts, measured in bytes.
+
+ |
```

Comment:
Indent looks incorrect compared to other options, please check and fix:
<img width="650" alt="image" src="https://github.com/user-attachments/assets/de60ff16-fa98-45ab-b54c-b4baab7af4be" />